caas/kubernetes/provider/modeloperator.go

```diff
@@ -100,6 +107,13 @@ func (m *modelOperatorBrokerBridge) Namespace() string {
 	return m.namespace()
 }
 
+func (m *modelOperatorBrokerBridge) IsLegacyLabels() bool {
+	if m.isLegacyLabels == nil {
+		return false
+	}
+	return m.isLegacyLabels()
```

Comment:
return m.isLageacyLabels != nil && m.isLegacyLabels()

?

Reply:
This is just a protection mechanism for if the func is not defined on the bridge. Open to better suggestions

Reply:
Why would this function not be defined?